usdoj-crt/beta-ada
# Related content component

Rationale:
1. User research has shown that users want multiple entry points for each page. The related content component allows users to see multiple pages thematically related to the one they are currently on.

Changes:
1. A new component has been created in the `_includes/` folder called `related-content.html`
2. This component is based off of the USWDS [Summary Box component](https://designsystem.digital.gov/components/summary-box/)
3. To use it, it needs to be declared in the frontmatter of the page it will be one using `related-content: true`
4. Tags need to be specified in an array within the front matter of pages and on the page it will be used on. `tags: [tag1, tag2, tag3]`
5. Tags also need to be whitelisted in a .yml file in the `_data/` directory. The file `tags.yml` contains a list of allowed tags that needs to contain the tags you have in your page's frontmatter. 

Tests:
1. @chinelo-ikejimba We need to see what this will look like on mobile. 
2. Check on multiple browsers
3. Check on mobile devices
4. Try adding news tags without adding to the white list
5. Make sure all pages with appropriate tags are listed
